tests/integration/connector.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import mysql.connector
+
+
+class MysqlConnector:
```

Comment:
I like contexts a lot!!! Well done! Beside when using db, I really like the Unit of Work (UoW) approach. :+1: